libs/bixarena/api-description/src/components/schemas/VoteCreateRequest.yaml

```diff
@@ -0,0 +1,10 @@
+type: object
+description: The information used to create a new vote.
+properties:
+  battleId:
+    $ref: BattleId.yaml
+  preference:
+    $ref: VotePreference.yaml
+required:
+  - battleId
+  - preference
```

Comment:
This object could be removed if all the parameters to vote are passed as query parameters. E.g. `POST /battles/{battleId}/vote/{modelA}`